virttest/qemu_devices/qcontainer.py

```diff
@@ -1756,9 +1771,9 @@ def pcic_by_params(self, name, params):
         :warning: x3130-upstream device creates only x3130-upstream device
                   and you are responsible for creating the downstream ports.
         """
-        driver = params.get('type', 'ioh3420')
+        driver = params.get('type', 'pcie-root-port')
         pcic_params = {"id": name}
-        if driver in ('ioh3420', 'x3130-upstream', 'x3130'):
+        if driver in self.root_type_list() or ('x3130-upstream', 'x3130'):
```

Comment:
This is probably not what you had in mind as it'll resolut in `bus_type` always to be `PCIE`.

Reply:
@ldoktor Yes...thanks for your patience. Updated.